cerberus/kubernetes/client.py

```diff
@@ -78,6 +78,32 @@ def list_namespaces():
     return namespaces
 
 
+# Monitor the status of all specified namespaces
+# and set the status to true or false
+def monitor_namespaces_status(watch_namespaces, watch_terminating_namespaces, iteration, iter_track_time):
+    namespaces = []
+    none_terminating = True
+    if watch_terminating_namespaces:
+        watch_nodes_start_time = time.time()
+        try:
+            ret = cli.list_namespace(pretty=True)
+        except ApiException as e:
+            logging.error("Exception when calling CoreV1Api->list_namespaced_pod: %s\n" % e)
```

Comment:
The process here continues after logging the error with the assumption that `ret` is set. The entire block should be wrapped in the `try-except`or the error should be handled.